deploy/helm/charts/values.yaml

```diff
@@ -33,7 +33,7 @@ lvmNode:
     type: RollingUpdate
   annotations: {}
   podAnnotations: {}
-  # This can be configured to run on various different k8s ditributions like
+  # This can be configured to run on various different k8s distributions like
```

Comment:
This will require a chart version bump as well in Chart.yaml file.

Reply:
Done!